docs/services/education-consultant.md

```diff
@@ -0,0 +1,31 @@
+# Education Consultant
+
+The Education Consultant has experience with designing curriculum and creating positive learning spaces for data science education.
+
+## Goals
```

Comment:
Here are some reworded goals getting a little closer to framing things in terms of learning outcomes and effectiveness. Let me know if you think this covers it, or there is more to articulate:

* Helping all participants feel able to intellectually engage in the educational and project spaces at hackweeks.
* Helping tutorial leads build content that is effective for participants across a wide range of backgrounds and experience levels.